### PR TITLE
crm_claim_rma_code: migration V9

### PR DESCRIPTION
**Pull request by cyrilgdn**
Depends on:
- [x] https://github.com/OCA/crm/pull/90 
- [x] https://github.com/OCA/crm/pull/92

### CODE REVIEW

File: crm_claim_rma_code/README.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ Bug Tracker
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/rma/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/rma/issues/new?body=module:%20crm_claim_rma_code%0Aversion:%208.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`here <https://github.com/OCA/rma/issues/new?body=module:%20crm_claim_rma_code%0Aversion:%209.0.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (yvaucher) [inline crm_claim_rma_code/README.rst]: This link can be removed. The README.rst template has been simplyfied see: 
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: crm_claim_rma_code/models/__init__.py [hunk 2]
```diff
@@ -1,22 +1,5 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2015 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (yvaucher) [inline crm_claim_rma_code/models/__init__.py]: License can completely be removed from `__init__.py` as the code is not there

**Comment by guewen** [comment]
:+1:

File: crm_claim_rma_code/__openerp__.py [hunk 3]
```diff
@@ -35,5 +18,5 @@
         'views/crm_claim_type.xml',
     ],
     'active': True,
```
Comment (moylop260) [inline crm_claim_rma_code/__openerp__.py]: crm_claim_rma_code/**openerp**.py:5: [C8103(manifest-deprecated-key), ] Deprecated key "active" in manifest file

**Comment by moylop260** [comment]
👍

**Comment by mdietrichc2c** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**